modules/ROOT/pages/index.adoc

```diff
@@ -20,6 +20,49 @@ With Mule you can:
 * Use error constructs and try scopes for rapid debugging.
 * Isolate the classloader to protect Mule apps from changes to the runtime or connectors.
 
+[[install_deploy_manage]]
+== Installation, Deployment, Management
+
+For a Mule app to run, it must be deployed into an environment where the Mule
+runtime engine is installed.
+
+Mule is packaged within the xref:7.3@studio::index.adoc[Studio] IDE and with
+xref:design-center::index.adoc[Design Center] on Anypoint Platform so
```

Comment:
"on the Anypoint Platform"

Reply:
we usually don't use an article before "Anypoint Platform"

Reply:
examples here: https://docs.mulesoft.com/general/